File: src/main/java/com/koddy/server/member/domain/repository/query/response/AppliedCoffeeChatsByMentee.java
package com.koddy.server.member.domain.repository.query.response;

import com.koddy.server.member.domain.model.Nationality;
import com.koddy.server.member.domain.model.mentee.Interest;
import com.querydsl.core.annotations.QueryProjection;

public record AppliedCoffeeChatsByMentee(
        long coffeeChatId,
        long menteeId,
        String name,
        String profileImageUrl,
        Nationality nationality,
        Interest interest
) {
    @QueryProjection
    public AppliedCoffeeChatsByMentee {
    }
}

File: src/main/kotlin/com/koddy/server/member/domain/repository/query/response/AppliedCoffeeChatsByMentee.kt
package com.koddy.server.member.domain.repository.query.response

import com.koddy.server.member.domain.model.Nationality
import com.koddy.server.member.domain.model.mentee.Interest

data class AppliedCoffeeChatsByMentee(
    val coffeeChatId: Long,
    val menteeId: Long,
    val name: String,
    val profileImageUrl: String?,
    val nationality: Nationality,
    val interest: Interest,
)
